app/frontend/src/pages/ask/Ask.tsx

```diff
@@ -259,6 +279,30 @@ export function Component(): JSX.Element {
                     snapToStep
                 />
 
+                <Slider
+                    className={styles.chatSettingsSeparator}
+                    label="Min Search Score"
+                    min={0}
+                    max={1}
+                    step={0.1}
+                    defaultValue={minSearchScore}
+                    onChange={onMinSearchScoreChange}
+                    showValue
+                    snapToStep
+                />
```

Comment:
I wonder if 0.1/0.2/0.3 is detailed enough, given it only goes from 0 to 1? More importantly, as far as I understand, this score varies based on whether the mode is vector, hybrid, or text:
https://learn.microsoft.com/en-us/azure/search/hybrid-search-ranking#scores-in-a-hybrid-search-results
So I think it'd have to be an `input type=number` or whatever the equivalent FluentUI component is.

Please correct me if I'm wrong though.

Reply:
Yes, you're right. The score range varies significantly depending on the search method. I've updated the UI to replace the slider with a numeric input field.